components/src/molecules/ParametersTable/__tests__/NoParameters.test.tsx

```diff
@@ -36,6 +36,6 @@ describe('NoParameters', () => {
     render({
       t: tMock,
     })
-    screen.getByText('no_parameters')
+    screen.getByText('no_parameters_specified')
```

Comment:
I think we would need to check the actual text here.

```suggestion
    screen.getByText('No parameters specified in this protocol')
```